pkg/ddl/table.go

```diff
@@ -760,7 +760,8 @@ func onRenameTable(jobCtx *jobContext, t *meta.Meta, job *model.Job) (ver int64,
 		return ver, errors.Trace(err)
 	}
 	oldTableName := tblInfo.Name
-	ver, err = checkAndRenameTables(t, job, tblInfo, oldSchemaID, job.SchemaID, &oldSchemaName, &tableName)
+	args.NewSchemaID = job.SchemaID
```

Comment:
why change it?

if we reach here, we are in first step, either first time or retry on error/failover, then the args.NewSchemaID is correct

Reply:
can you set it in GetRenameTableArgs, and add a comment that it's not set for rename-table

Reply:
add some comment is ok for now